website/blog/2020-02-31-version-60.md

```diff
@@ -0,0 +1,61 @@
+
+## Native Modules are now Autolinked
+
+The team working on the [React Native CLI](https://github.com/react-native-community/cli) has introduced major improvements to native module linking called [autolinking](https://github.com/react-native-community/cli/blob/master/docs/autolinking.md)! Most scenarios will not require the use of `react-native link` anymore. At the same time, the team overhauled the linking process in general. Let us know what you think using the hashtag [#autolinkingInRN](https://twitter.com/intent/tweet?hashtags=autolinkingInRN).
+
+## Upgrade Helper
+
+[@lucasbento](https://github.com/lucasbento), [@pvinis](https://github.com/pvinis), [@kelset](https://github.com/kelset), and [@watadarkstar](https://github.com/watadarkstar) have built a great tool called [Upgrade Helper](https://react-native-community.github.io/upgrade-helper/) to make the upgrade process simpler. It helps React Native users with brownfield apps or complex customizations to see what's changed between versions. Take a look at the [upgrade from 0.59.9 to 0.60.0](https://react-native-community.github.io/upgrade-helper/?from=0.59.9&to=0.60.0) and try it out today for your upgrade path!
+
+![Upgrade Helper cleanly and easily shows the changes needed to migrate to a different version of React Native](/react-native/blog/assets/0.60-upgrade-helper.png)
+
+## A Note to Library Maintainers
+
+Changes for AndroidX will almost certainly require updates to your library, so be sure to include support soon. If you're not able to upgrade yet, consider checking your library against the jetifier to confirm that users are able to patch your library at build time.
+
+Depending on how your library was previously integrated, you may also need to make some changes. Check the [dependencies](https://github.com/react-native-community/cli/blob/master/docs/dependencies.md) guide from the CLI for information on how to define your dependency interface.
+
+## Thanks
+
+While these are the highlights that we noted, there are many others to be excited about. To see all the updates, take a look at the [changelog](https://github.com/react-native-community/react-native-releases/blob/master/CHANGELOG.md). As always, stay tuned for more exciting changes in the future. Enjoy 0.60 in the meantime.
+
+[Ryan](https://github.com/turnrye) and the whole [React Native core team](https://twitter.com/reactnative)
```

Comment:
```suggestion
Remember that you can always also follow us on twitter: [React Native core team](https://twitter.com/reactnative) & [React Native Community](https://twitter.com/ReactNativeComm)
```

Reply:
(also, I don't think it's necessary to have your signature here since you'll show on top, right below the title)